src/shared/ui/Accordion/Accordion.tsx

```
@@ -1,70 +1,125 @@
-      </AccordionPrimitives.Header>
-      <AccordionPrimitives.Content
-        asChild
-        className="data-[state=closed]:animate-accordion-up data-[state=open]:animate-accordion-down overflow-hidden px-6 py-[17px]"
+    <div className="border-b border-gray-200" {...props}>
+      <button
+        onClick={() => toggleItem(value)}
+        className={cn(
+          'flex w-full cursor-pointer items-center justify-between px-6 py-4 text-left hover:bg-gray-50',
+          btnClassName
+        )}
       >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)